collections/business/general-notifications/event-general-notifications/phone-number-deleted/event-general-notifications.json

```diff
@@ -0,0 +1,29 @@
+{
+  "eventClassName": "com.backbase.dbs.user.manager.event.spec.v2.DeletePhoneNumberSecNotEvent",
+  "id": "phone-number-deleted",
+  "eventBusinessName": "Phone Number Deleted",
+  "recipientType": "internal",
+  "extractor": "#this['userInternalId']",
+  "generalNotifications": [
+    {
+      "id": "phone-number-deleted",
+      "name": "Phone number deleted",
+      "userPreferenceEnabled": false,
+      "deliveryOptions": [
+        {
+          "channel": "email",
+          "channelSettings": "/phone-number-deleted/email/email_channel-settings.json",
+          "isEnabledByDefault": true,
+          "isManageableByUser": false,
+          "engagementTemplates": [
+            {
+              "path": "/phone-number-deleted/email/en_US/engagement-template.json",
+              "templateName": "phone-number-deleted-email",
```

Comment:
Hi 👋, I was reviewing another pull request about the same topic in another repository and all the comments apply to this one too, please can you take a look?

TL;DR: For email we don't have manageability via Engage App thus we don't need to inform templateName, template.json, content-schema.json and placeholders.json. Moreover there is an embedded way to specify the html file directly here :)